google/cloud/language/v1beta1/language_gapic.yaml

```diff
@@ -3,7 +3,7 @@ language_settings:
   java:
     package_name: com.google.cloud.language.spi.v1beta1
   python:
-    package_name: google.cloud.language.v1beta1
```

Comment:
Why are you putting the generated packages under the GAX namespace? GAX = Google Api eXtensions, which is supposed to be exclusively the API-agnostic run-time library.